deploy/manifests/crds

```diff
@@ -0,0 +1 @@
+../../config/crd/bases
```

Comment:
I've decided to add a symlink in the manifests too to minimize the changes, this will probably go away in a future PR